### ISSUE TITLE
Search Select Item color hard to see in dark mode

### ISSUE THREAD

**Issue by yuanLeeMidori** [Seneca-CDOT/telescope#2028]
**What happened**:
![Screenshot_2021-03-25 Search Telescope(1)](https://user-images.githubusercontent.com/50813726/112560560-e1aaf900-8da9-11eb-8158-0b1af75cc114.png)
The font color of search type option is hard to see in dark mode.
**What should have happened**:

**How to reproduce it (as precise as possible)**:

**Anything else we need to know?**:

**Environment**:

- OS:
- Browser:

**Comment by PedroFonsecaDEV** [comment]
Closed: #2029

**Comment by yuanLeeMidori** [comment]
Hi, @PedroFonsecaDEV 
I didn't make it in #2029 (I thought I did but I didn't) , I'm gonna reopen this issue, thanks!

**Comment by yuanLeeMidori** [comment]
Search bar will be redesign in #2005
